ntc_templates/templates/hp_comware_display_link-aggregation_verbose.textfsm

```diff
@@ -0,0 +1,30 @@
+Value Required INTERFACE (\S+)
+Value CREATION_MODE (.+)
+Value AGGREGATION_MODE (.+)
+Value LOADSHARING (.+)
+Value List LOCAL_INTERFACES (\S+)
+Value List REMOTE_INTERFACES (\S+)
+
+Start
+  ^Aggregat\S+\s+Interface:\s+${INTERFACE}
+  ^Creation\s+Mode:\s+${CREATION_MODE}
+  ^Aggregation\s+Mode:\s+${AGGREGATION_MODE}
+  ^Loadsharing\s+Type:\s+${LOADSHARING}
+  ^Local: -> LocalPorts
```

Comment:
Add ```  ^\s*$$
  ^. -> Error```

to all 3 states, (Start, LocalPorts, RemotePorts)

Reply:
@itdependsnetworks fixed, thank you